src/abilities/Snow-Bunny.js

```diff
@@ -340,7 +342,7 @@ export default G => {
 				let dist = projectileInstance[2];
 
 				tween.onComplete.add(function() {
```

Comment:
Not updating to arrow methods or leaving for future cleanup? If leaving can we toggle a config to easily highlight them later?

Reply:
i'm not making changes unless it's required by the linter